src/components/TxDetailsModal.js

```diff
@@ -34,6 +35,14 @@ class TxDetailsModal extends Component {
     const idStr = getShortHash(tx.txId, 12);
     const explorerIcon = <Image source={icShareActive} width={24} height={24} />;
     const explorerLink = `https://explorer.hathor.network/transaction/${tx.txId}`;
+    const txIdComponent = (
+      <CopyClipboard
+        text={idStr}
+        copyText={tx.txId}
+        copiedTimeout={800}
```

Comment:
Why did you change the default? It's a bit fast from the video you shared. The default is used in the wallet copy component, and I think we could keep it the same.

Reply:
Seemed to slow for me, especially when the modal might be closed. In that case, we get a warning of changing the state of an unmounted component.

Reply:
The warning is not related to the slowness, the event listener must be removed in the unmount of the component.

I still feel 1500ms is better but I'm ok to move with 800ms. 

However we need to remove the listeners anyway.

Reply:
In this case, I imagine it comes from the `setTimeout` on `src/components/CopyClipboard.js`. There's no listener to remove on `TxDetailsModal.js`.

Reply:
Yeah, you need to get the `setTimeout` reference and clear it in the `componentUnmount`

Reply:
Fixed and went back to default timeout.